### PR TITLE
feat: sidebar

### PR DESCRIPTION
**Pull request by mxrguspxrtwork**

### CODE REVIEW

**Review by tkullisaar — CHANGES_REQUESTED** [review]
In order to add component into Moon DS component library, we kindly request that you follow our [contributing guidelines](https://github.com/coingaming/moon/blob/main/CONTRIBUTING.md). For new component inclusion, we kindly ask you to showcase its usage either through examples or on the site. Additionally, we kindly request that every property is accompanied by a @doc attribute.

File: assets/css/app.css [hunk 1]
```diff
@@ -229,4 +229,10 @@ input:-webkit-autofill:focus {
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
   font-weight: 500;
   src: local("AvertaStd-Semibold"), local("Averta Std Semibold"), url(https://sportsbet.io/sportsbet-io/files/fonts/averta-std/semibold.woff2) format("woff2");
-}
\ No newline at end of file
+}
+
+@layer components {
```
Comment (ischepin) [inline assets/css/app.css:234]: should this remain here? there is no sidebar now

**Comment by phgrey** [comment]
fix: tooltip.trigger fix

File: assets/css/app.css (hunk 1 above)
Comment (mxrguspxrtwork) [reply assets/css/app.css:234]: good notice!

**Pull request merged**